R/Reactant/build_tarballs.jl

```diff
@@ -348,12 +368,19 @@ augment_platform_block="""
             # so that we get recompiled if the driver changes.
             if cuname != "" && gpu == "undecided"
                 handle = Libdl.dlopen(cuname)
+                cuda_version = cuDriverGetVersion(handle)
                 path = Libdl.dlpath(handle)
                 Libdl.dlclose(handle)
 
-                @debug "Adding include dependency on \$path"
-                Base.include_dependency(path)
-                gpu = "cuda"
+                if cuda_version isa VersionNumber
+                    if cuda_version == v"12.6"
```

Comment:
I don't know what check you exactly want, but you can tweak it here.

Reply:
I think probably >= 12.6

We can also build different reactant versions for different cuda versions too